### PR TITLE
Add the ability to register as a mentor on the mentoring confir…

### PR DESCRIPTION
**Pull request by Pulya10c**
…mation page

**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [ ] I followed naming convention rules

---

#### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

https://github.com/rolling-scopes/rsschool-app/issues/1771

#### 💡 Background and solution

_Describe the big picture of your changes here_

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Database migration is added or not needed
- [ ] Documentation is updated/provided or not needed
- [ ] Changes are tested locally

### CODE REVIEW

File: client/src/components/Warning/Warning.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+type Props = {
+  loading: boolean;
+  githubId: string;
+  imagePath: string;
+  nameImage: string;
+  textMessage: string | JSX.Element;
+};
+
+export function Warning(props: Props) {
+  const { loading = false, githubId, imagePath, nameImage = defaultImageName, textMessage = defaultMessage } = props;
```
Comment (AlreadyBored) [inline client/src/components/Warning/Warning.tsx:17]: Please consider to choose 1 option: optional props & default values || mandatory props & no default values :)

File: client/src/components/Warning/Warning.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  return (
+    <PageLayout loading={loading} githubId={githubId}>
+      <Row justify="center" style={{ margin: '65px 0 25px 0' }}>
+        <Image src={`/static${imagePath}`} alt={nameImage} width={175} height={175} />
```
Comment (AlreadyBored) [inline client/src/components/Warning/Warning.tsx:21]: `nameImage`=> `imageName`?

Comment (Pulya10c) [reply client/src/components/Warning/Warning.tsx]: fixed, thanks

File: client/src/components/Warning/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export { Warning } from './Warning';
```
Comment (AlreadyBored) [inline client/src/components/Warning/index.ts:1]: Minor one: if you plan to have only one file with code file in this folder, then you can put it directly in `index.tsx` and remove `index.ts`

**Review by AlreadyBored — APPROVED** [review]

File: client/src/components/Warning/index.ts (hunk 3 above)
Comment (Pulya10c) [reply client/src/components/Warning/index.ts]: fixed

**Pull request merged**